internal/common/address.go

```diff
@@ -52,6 +61,16 @@ func IsBech32Address(s string) bool {
 	return true
 }
 
+// IsEntryPointAddress returns true if the Address is the ENTRY_POINT Address.
+func (a Address) IsEntryPoint() bool {
```

Comment:
why not just hardcode the entry point address and do byte.compare?

Reply:
this is taken directly from the upstream implementation. does it make any difference?